app-emulation/virtio-win/virtio-win-0.1.221.1.ebuild

```
@@ -10,7 +10,7 @@ DESCRIPTION="VirtIO drivers for Windows virtual machines running on KVM"
 HOMEPAGE="https://docs.fedoraproject.org/en-US/quick-docs/creating-windows-virtual-machines-using-virtio-drivers/index.html"
 SRC_URI="https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/archive-virtio/${PN}-${MY_DW}/${PN}-${MY_PV}.iso"
 
-LICENSE="GPL-2"
+LICENSE="BSD"
```

License is not BSD, see [bug 892037](https://bugs.gentoo.org/892037).
```suggestion
LICENSE="GPL-2 GPL-2+ GPL-3+ LGPL-2+ Apache-2.0 BSD MS-RL"
```
(Same for the other ebuilds, of course.)

MS-RL would have to be added to `licenses/`. It can be found here: https://directory.fsf.org/wiki/License:MS-RL

---

It's fixed now. Thank you both. If there're things that need to change, please let me know.